### PR TITLE
Don't choke on undefined options.

### LINKED ISSUE #36: TypeError: Cannot read property 'id' of undefined at deprecationCollector
I think [this change](https://github.com/mixonic/ember-cli-deprecation-workflow/commit/36b7e599d35c428cb4465b505cafb74abcebba9d) (0.2.0 to 0.2.1) may have broken my build...

```
TypeError: Cannot read property 'id' of undefined at deprecationCollector
```

I have an existing `config/deprecation-workflow.js` for an Ember v1.13.12 and Ember Data v1.13.16 project which was populated with a substantial `matchMessage` array.  I believe it was parsing a `Ember.SortableMixin is deprecated and was used in...` message and attempted a `var create = Object.create || Ember.create;` just before the exception.

Reverting to v0.2.0 appears to silence the error.

```
ember version
version: 1.13.15
node: 4.4.2
npm: 3.8.5
os: darwin x64
```

### PR DESCRIPTION
**Pull request by kayakyakr**
Options sometimes being passed as undefined. Don't let it.

Fixes #36

### CODE REVIEW

**Comment by rwjblue** [comment]
:+1:

File: vendor/ember-cli-deprecation-workflow/main.js [hunk 1]
```diff
@@ -54,7 +54,7 @@
   });
 
   Ember.Debug.registerDeprecationHandler(function deprecationCollector(message, options, next){
-    var key = options.id || message;
+    var key = options && options.id || message;
     var matchKey = key === options.id ? 'matchId' : 'matchMessage';
```
Comment (rwjblue) [inline vendor/ember-cli-deprecation-workflow/main.js]: this also needs to be updated

Comment (kayakyakr) [inline vendor/ember-cli-deprecation-workflow/main.js]: Yes it does indeed. Yay for failing tests?

**Comment by kayakyakr** [comment]
This is ready to go. Any chance we see 0.2.2 tonight?

**Comment by rwjblue** [comment]
Ya, gimme a few, will release once I get back to my laptop.

**Comment by rwjblue** [comment]
Thanks for submitting the fix!

**Pull request merged**